.gitignore

```diff
@@ -5,6 +5,7 @@ build
 .opt-in
 .opt-out
 .DS_Store
+.idea
```

Comment:
This should be in your [global gitignore file](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer). But I'm fine including it here too.